examples/chip-tool/commands/pairing/Commands.h

```diff
@@ -157,31 +175,31 @@ class StartUdcServerCommand : public CHIPCommand
     }
 };
 
-void registerCommandsPairing(Commands & commands)
+void registerCommandsPairing(Commands & commands, CredentialIssuerCommands * credIssuerCommands)
 {
     const char * clusterName = "Pairing";
 
     commands_list clusterCommands = {
-        make_unique<Unpair>(),
-        make_unique<PairQRCode>(),
-        make_unique<PairManualCode>(),
-        make_unique<PairBleWiFi>(),
-        make_unique<PairBleThread>(),
-        make_unique<PairSoftAP>(),
-        make_unique<Ethernet>(),
-        make_unique<PairOnNetwork>(),
-        make_unique<PairOnNetworkShort>(),
-        make_unique<PairOnNetworkLong>(),
-        make_unique<PairOnNetworkVendor>(),
-        make_unique<PairOnNetworkCommissioningMode>(),
-        make_unique<PairOnNetworkCommissioner>(),
-        make_unique<PairOnNetworkDeviceType>(),
-        make_unique<PairOnNetworkDeviceType>(),
-        make_unique<PairOnNetworkInstanceName>(),
+        make_unique<Unpair>(credIssuerCommands),
```

Comment:
A better name than `credIssuerCommands` would be `credentialsConfig`

Reply:
Addressed in latest commit